package.json

```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "build": "tsc",
     "lint": "eslint src",
+    "docs": "asciidoctor -D build/ docs/index.adoc"
```

Comment:
needs a comma on the end of the line? (I don't see one) not valid json like this?

Reply:
thanks, doh bad merge conflict 🤦‍♂️ - CI failed then I realised the issue